MeshLib/Elements/MapBulkElementPoint.h

```diff
@@ -17,6 +17,18 @@
 
 namespace MeshLib
 {
+/// Maps the given lower dimensional boundary point \c wp of a line, i.e. the 1d
+/// gauss point given in local coordinates of a line, to higher dimensional
+/// point of the tri face (defined by the tri element and the face id) also in
+/// local coordinates of the tri face.
+/// \param tri the tri element
+/// \param face_id the id of the tri face the point will be mapped on
+/// \param wp the gauss point of the lower dimensional element
```

Comment:
Minor; Would call it integration point at least.
Better would be to call it something like local coordinate of the line element. (and then to pass only the coordinates w/o the integration weight.) But this is a long story...